todo-app/src/Components/todoList.js

```diff
@@ -0,0 +1,57 @@
+//jshint esnext: true
+
+import React from 'react';
+
+import TodoItem from './todoItem';
+import todosData from './todos.json';
+import './styles/todoList.css';
+
+const todos = todosData.todos;
+
+
+export default class TodoList extends React.Component {
+
+    state = {
+        todos,
+    }
+
+    handleIsItDone = (todoItem) => {
```

Comment:
It's probably better to call this function something like `handleToggleDone` or something, so that it's more obvious what the function actually does. With the current name, it makes me think the function just _checks_ if the thing is done, not actually changes it.